modules/test_user_dashboard/spec/requests/tud_accounts_request_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Test User Dashboard', type: :request do
+  context 'index' do
+    it 'returns a json formatted respone' do
```

Comment:
Again this might be a me thing:

```ruby
context 'with json' do 
  before { get '/test_user_dashboard/tud_accounts') }
  
  subject :response

  it {  expect(response.status).to eq 200 }
  it { expect(response.content_type).to eq 'application/json; charset=utf-8' }
end
```

I'm pretty particular about only 1 expectation per it block. 

I tend to follow any recommendation from Better Specs:

https://www.betterspecs.org/#single

Reply:
Agreed Jim! I think it's a personal preference, but one per block definitely reads better